lib/assertions/match-json.test.js

```diff
@@ -1,77 +1,50 @@
 "use strict";
 
-var testHelper = require("../test-helper");
+var assert = require("assert");
+var referee = require("../referee");
 
-testHelper.assertionTests("assert", "matchJson", function(
-    pass,
-    fail,
-    msg,
-    error
-) {
-    pass("when json string matches object", '{"key":"value","and":42}', {
-        key: "value"
+describe("matchJson", function() {
```

Comment:
I think this should read `assert.matchJson`

Reply:
Good spot, thanks 👍should now be corrected